### PR TITLE
Add two more sections to the template

### PR DESCRIPTION
**Pull request by bloated-devish**
Can we add three more sections to the template?

1) Alternate designs - Gives the community idea on why some design choices were skipped. 
2) Data storage change - As more fields are added, we need to start ensuring these new fields don't increase gas fees so much (as well as making nodes disk heavy).
3) Acknowledgements - People often have others who contributed, but not authors. This is a better place to ensure proper crediting?

### CODE REVIEW

**Comment by bloated-devish** [comment]
Let me start a CIP and the reopen this PR

**Comment by bloated-devish** [comment]
Based on discussion https://github.com/bitclout/cips/discussions/33, updated to two sections.

**Review by maebeam — APPROVED** [review]

**Comment by maebeam** [comment]
> I specifically added Data Storage - since we are growing data a lot - and its good for CIP creators and core devs to get an opportunity to think if this will bloat the blockchain. It's much harder to go and fix mistakes that caused duplicate data or large data increase that overall increases cost of every post (usually which is unintended).
>
> I'll go ahead and leave data storage out, but if you agree, I can follow up and add it as a sub section to specification (I agree it feels better to put it there), or just leave it out completely (although you'll hear bloated grumbling a bit lol).

If you want to update this PR to include a note about optimizing for data storage under Specification that would be great.

**Comment by bloated-devish** [comment]
> > I specifically added Data Storage - since we are growing data a lot - and its good for CIP creators and core devs to get an opportunity to think if this will bloat the blockchain. It's much harder to go and fix mistakes that caused duplicate data or large data increase that overall increases cost of every post (usually which is unintended).
> > I'll go ahead and leave data storage out, but if you agree, I can follow up and add it as a sub section to specification (I agree it feels better to put it there), or just leave it out completely (although you'll hear bloated grumbling a bit lol).
> 
> If you want to update this PR to include a note about optimizing for data storage under Specification that would be great.

Done! Thank you!

**Review by HPaulson — APPROVED** [review]

File: cip-template.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ Include an explanation of the "why" behind this CIP. What problem does it solve?
 The technical specification should describe the new feature in deep technical detail. It should be detailed enough to allow for complete implementation.
   
 Include the rationale behind any decisions. If there were any approaches that didn't work, feel free to explain them to help future devs understand why things are the way they are.
+ 
+### Data Storage Change
+Identify the additional storage requirement in the blockchain due to this change. Carefully consider if this CIP is creating additional duplication or if there are opportunity to optimize increase in size due to this CIP.
```
Comment (maebeam) [inline cip-template.md:27]: Can we fix the grammar a bit:

> Identify any additional storage requirements. Carefully consider if this CIP duplicates data or if there are opportunities to optimize the data model.

Comment (bloated-devish) [reply cip-template.md]: Done

**Pull request merged**